src/libraries/System.Private.DataContractSerialization/src/System/Runtime/Serialization/Json/JsonReaderWriterFactory.cs

```diff
@@ -69,7 +69,7 @@ public static XmlDictionaryWriter CreateJsonWriter(Stream stream, Encoding encod
             return CreateJsonWriter(stream, encoding, ownsStream, indent, JsonReaderWriterFactory.DefaultIndentChars);
         }
 
-        public static XmlDictionaryWriter CreateJsonWriter(Stream stream, Encoding encoding, bool ownsStream, bool indent, string indentChars)
+        public static XmlDictionaryWriter CreateJsonWriter(Stream stream, Encoding encoding, bool ownsStream, bool indent, string? indentChars)
```

Comment:
should encoding be nullable?

Reply:
Encoding can't be null, it is checked for null in `SetOutput`:

https://github.com/dotnet/runtime/blob/84340ed4cf32d9ed75943625f2000f7fed29e230/src/libraries/System.Private.DataContractSerialization/src/System/Runtime/Serialization/Json/XmlJsonWriter.cs#L254-L263

In general, I've seen that Encoding can be null when reading, but when writing you must pass a valid Encoding.